delta/plugins/elasticsearch/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/elasticsearch/indexing/GraphResourceToDocument.scala

```diff
@@ -33,8 +33,9 @@ final class GraphResourceToDocument(context: ContextValue, includeContext: Boole
 
   /** Given a [[GraphResource]] returns a JSON-LD created from the merged graph and metadata graph */
   def graphToDocument(element: GraphResource): Task[Option[Json]] = {
-    val graph = element.graph ++ element.metadataGraph
-    val json  =
+    implicit val jsonLdOptions = JsonLdOptions.AlwaysEmbed
```

Comment:
Do we need to set this here? Is it not possible to make it an argument to the whole class, so that we can set it to be `@always` only from the CompositeSink?

Reply:
Let me have a think about this one

Reply:
I did what you suggested and it works. I think given that this is the smallest possible scope of change, you're right